packages/website/tailwind.config.js

```diff
@@ -1,14 +1,39 @@
+const colors = require("tailwindcss/colors");
+
 /** @type {import('tailwindcss').Config} */
-export default {
-  content: ["./index.html", "./{src,routes}/**/*.{ts,tsx}"],
+module.exports = {
```

Comment:
Does this work? Everything should be ESM.
```suggestion
export default {
```